src/sentry/api/endpoints/project_performance_issue_settings.py

```diff
@@ -273,7 +291,7 @@ def put(self, request: Request, project) -> Response:
             {**performance_issue_settings_default, **performance_issue_settings, **data},
         )
 
-        if body_has_admin_options:
+        if body_has_admin_options or body_has_management_options:
             self.create_audit_entry(
```

Comment:
Even though its not a super user setting anymore, I think managing a detector should still have an audit log entry.